.ci/pytorch/test.sh

```diff
@@ -391,7 +392,21 @@ test_inductor_cpp_wrapper_abi_compatible() {
 # .github/workflows/inductor-perf-test-nightly.yml
 DYNAMO_BENCHMARK_FLAGS=()
 
-if [[ "${TEST_CONFIG}" == *dynamo_eager* ]]; then
+pr_time_benchmarks() {
+
+  pip install cirron
```

Comment:
Do you need to install cirron here as you already added it to requirements.txt ?

Reply:
When i did not have it here, it did failed with module not found